src/ontology/mondo-edit.obo

```diff
@@ -82425,7 +82424,6 @@ xref: MEDGEN:308434 {source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
 xref: MESH:D001170 {source="DOID:813", source="MONDO:equivalentTo"}
 xref: NANDO:2200437 {source="MONDO:NANDO", source="https://orcid.org/0000-0003-0011-764X", source="https://orcid.org/0000-0002-0170-9172"}
 xref: NCIT:C26699 {source="MONDO:equivalentTo"}
-xref: NCIT:C26700 {source="DOID:813", source="MONDO:directSiblingOf"}
```

Comment:
[NCIT:C26700](https://www.ebi.ac.uk/ols4/ontologies/ncit/classes?short_form=NCIT_C26700) is infective arthritis and is a parent of bacterial arthritis so this is not correct that NCIT:C26700 is a sibling of MONDO:0004471 'bacterial arthritis'.